src/main/java/com/nextstep/lotto/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,56 @@
+package com.nextstep.lotto.calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final int ZERO = 0;
+    private static final String CUSTOM_REGEX = "//(.)\n(.*)";
+
+    public static int splitAndSum(String source) {
```

Comment:
```suggestion
    public static int sum(String source) {
```
이 객체를 사용하는 클라이언트 입장에서는 어떤 기능을 하는지만 알 수 있으면 됩니다
split이라는 상세 내용을 외부에 알려줄 필요는 없어보이네요

Reply:
과제 요구사항에 테스트코드가 있길래 그걸 토대로 구현하라는 뜻인 줄 알았는데 아니었나보네요 ㅎㅎ